.github/workflows/book.yml

```diff
@@ -25,10 +25,12 @@ jobs:
     steps:
     # checkout source code
     - uses: actions/checkout@v3
-    # install rmarkdown
-    - name: Install rmarkdown
+    # install bookdown
+    - name: Install bookdown
       run: |
-        Rscript -e 'install.packages(c("rmarkdown","bookdown"))'
+        Rscript \
+          -e 'repos <- c(getOption("repos"), sub(r"(\d{4}-\d{2}-\d{2})", "latest", getOption("repos"))' \
```

Comment:
Borrowed from pecan.depends.R -- this makes install_version use the bookdown from the existing rocker-specified CRAN mirror if it's suitable, or the latest version if the Rocker-specified mirror is too old.